static_src/actions/upsi_actions.js

```diff
@@ -0,0 +1,75 @@
+import AppDispatcher from '../dispatcher';
+import { upsiActionTypes } from '../constants';
+import cfApi from '../util/cf_api';
+
+export const fetchAllSuccess = items => {
+  AppDispatcher.handleServerAction({
+    type: upsiActionTypes.UPSI_FETCH_ALL_SUCCESS,
+    items
+  });
+
+  return Promise.resolve({ items });
+};
+
+fetchAllSuccess.exportTest = true;
```

Comment:
what are these (this line and the other `exportTest`) needed for?

Reply:
I'll remove it for now. It's an idea I was toying with.

Rationale: testing JS modules often means you need to export certain things so that the test file can import them to test. Unlike in Go where it can test both exported and unexported members, with JS modules, we're forced to export all the things if we want to test them. However, I don't want callers to use these. E.g. `fetchAllSuccess` should never be directly called. So I toyed with the idea of setting `exportTest = true` as an indicator that it shouldn't be used. Of course, to actually enforce it we would need tooling: e.g. maybe eslint can error when it's imported and used. Also even though the bandwidth would be minimal, we could also use a babel plugin to strip them out. I didn't really look for existing solutions to this - I just made this up as I was writing the test.

Anyway, I'll just remove it for now, but it's something that we could revisit later out of interest.

Reply:
OK @jcscottiii removed.